editor.planx.uk/src/@planx/components/Send/uniform/UniformPayload/model.ts

```diff
@@ -178,32 +189,40 @@ export class UniformPayload implements IUniformPayload {
     },
   });
 
+  private getApplicant = (): PartialDeep<ApplicantOrAgent> => {
+    return this.getApplicantOrAgent("applicant");
+  };
+
+  private getAgent = (): PartialDeep<ApplicantOrAgent> | undefined => {
+    const isAgentInPassport = Boolean(
+      this.passport.data?.["applicant.agent.name.first"]
+    );
+    if (!isAgentInPassport) return;
+    return this.getApplicantOrAgent("applicant.agent");
+  };
+
   private getApplicantOrAgent = (
     person: "applicant.agent" | "applicant"
-  ): ApplicantOrAgent => ({
-    "common:PersonName": {
-      "pdt:PersonNameTitle": this.passport.data?.[`${person}.title`],
-      "pdt:PersonGivenName": this.passport.data?.[`${person}.name.first`],
-      "pdt:PersonFamilyName": this.passport.data?.[`${person}.name.last`],
-    },
-    "common:OrgName": this.passport.data?.[`${person}.company.name`],
-    "common:ContactDetails": {
-      "common:Email": {
-        "apd:EmailAddress": this.passport.data?.[`${person}.email`],
-        _EmailUsage: "work",
-        _EmailPreferred: "yes",
+  ): PartialDeep<ApplicantOrAgent> => {
+    return {
+      "common:PersonName": {
+        "pdt:PersonNameTitle": this.passport.data?.[`${person}.title`],
+        "pdt:PersonGivenName": this.passport.data?.[`${person}.name.first`],
+        "pdt:PersonFamilyName": this.passport.data?.[`${person}.name.last`],
       },
-      "common:Telephone": {
-        "apd:TelNationalNumber":
-          this.passport.data?.[`${person}.phone.primary`],
-        _TelUse: "work",
-        _TelPreferred: "no",
-        _TelMobile: "yes",
+      "common:OrgName": this.passport.data?.[`${person}.company.name`],
+      "common:ContactDetails": {
+        "common:Email": {
+          "apd:EmailAddress": this.passport.data?.[`${person}.email`],
+        },
+        "common:Telephone": {
+          "apd:TelNationalNumber":
+            this.passport.data?.[`${person}.phone.primary`],
+        },
```

Comment:
nice to see that defaults like `_TelUse`, `_TelPreferred`, and `_TelMobile` are defined on the ZOD schema now, that feels a lot more readability :+1: